keyboards/boardsource/equals/equals.c

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 @boardsource
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+void ui_init(void);
+void ui_task(void);
```

Comment:
It's erroring out on every keymap without the ui functions.  This would be the better way to handle these: 
```suggestion
__attribute__((weak)) void ui_init(void) {};
__attribute__((weak)) void ui_task(void) {};
```